logview/src/logview-log.c

```diff
@@ -654,8 +653,15 @@ log_load (GIOSchedulerJob *io_job,
   }
 
   log->priv->file_size = g_file_info_get_size (info);
-  g_file_info_get_modification_time (info, &timeval);
-  log->priv->file_time = timeval.tv_sec;
+  #if GLIB_CHECK_VERSION(2,61,2)
+    GDateTime *file_dt;
+    log->priv->file_time = g_file_info_get_modification_date_time (info);
```

Comment:
`log->priv->file_time` is `time_t` and [g_file_info_get_modification_date_time](https://developer.gnome.org/gio/stable/GFileInfo.html#g-file-info-get-modification-date-time) returns `GDateTime *`.
https://github.com/mate-desktop/mate-utils/blob/64f3d394222586ab9d98116e29f6479f9129b686/logview/src/logview-log.c#L49

Reply:
Use [g_date_time_to_unix](https://developer.gnome.org/glib/stable/glib-GDateTime.html#g-date-time-to-unix) to convert `GDateTime` object to Unix time.

Reply:
Example:
```C
/* gcc -Wall `pkg-config --cflags --libs glib-2.0` -o time time.c  */
#include <glib.h>
#include <stdio.h>
#include <time.h>

int main() {
	GDateTime *dt;
	gint64 t;

	dt = g_date_time_new_now_local ();
	/* Number of seconds that have elapsed since 1970-01-01 00:00:00 UTC */
	t = g_date_time_to_unix (dt);
	g_date_time_unref (dt);

	printf ("%" G_GINT64_FORMAT "\n%s\n", t, ctime(&t));

	return 0;
}
```

Reply:
Ok.

Thanks you very much for for your help.

Regards